packages/usa-input-prefix-suffix/src/usa-input-prefix-suffix.stories.js

```diff
@@ -3,6 +3,14 @@ import suffix from "./usa-input-suffix.twig";
 
 export default {
   title: "Components/Form Inputs/Input Prefix or Suffix",
+  argTypes: {
+    state: {
+      name: "State",
+      control: "radio",
+      options: ["default", "error", "disabled", "aria_disabled"],
+      defaultValue: "default"
```

Comment:
This control is called `state` while others are called `disabled_state` and this includes an error state option. Is there something specific about the input prefix/suffix error state that it needs to shown in this component? Wondering if it is safe to remove and just rely on the text input component for checking error/success states.

Also, I noticed there are no disabled/aria-disabled storiesfor this component.

```suggestion
    disabled_state: {
      name: "Disabled state",
      control: { type: "radio" },
      options: ["none", "disabled", "aria_disabled"],
      defaultValue: "none"
    },
```

Reply:
Resolved in [687156b](https://github.com/uswds/uswds/pull/5063/commits/687156b0f3418558f3d26662cba65c522a01ebe2)